server/neptune/temporalworker/server.go

```diff
@@ -246,6 +251,33 @@ func (s Server) Start() error {
 	return nil
 }
 
+func (s Server) buildDeployWorker() worker.Worker {
+	// pass the underlying client otherwise this will panic()
+	deployWorker := worker.New(s.TemporalClient.Client, workflows.DeployTaskQueue, worker.Options{
+		EnableSessionWorker: true,
+		WorkerStopTimeout:   TemporalWorkerTimeout,
+	})
+	deployWorker.RegisterActivity(s.DeployActivities)
+	deployWorker.RegisterActivity(s.GithubActivities)
+	deployWorker.RegisterActivity(s.TerraformActivities)
```

Comment:
remind me why we need terraform activities here?

Reply:
see prev comment: https://github.com/lyft/atlantis/pull/426#discussion_r1002848185, `GetWorkerInfo` is a "terraform" workflow activity that needs to be queued by the deploy taskqueue.